client_test.go

```diff
@@ -437,3 +455,38 @@ func TestWrapGrpcError(t *testing.T) {
 		})
 	}
 }
+
+// TestFetchSwapsLastHop asserts that FetchSwaps loads LastHop for LoopIn's.
+func TestFetchSwapsLastHop(t *testing.T) {
+	defer test.Guard(t)()
+
+	ctx := createClientTestContext(t, nil)
+
+	lastHop := route.Vertex{1, 2, 3}
+
+	// Create a loop in swap.
+	swapHash := lntypes.Hash{1, 1, 1}
+	swap := &loopdb.LoopInContract{
+		SwapContract: loopdb.SwapContract{
+			CltvExpiry:      111,
+			AmountRequested: 111,
+			ProtocolVersion: loopdb.ProtocolVersionMuSig2,
+			HtlcKeys:        htlcKeys,
+		},
+		LastHop: &lastHop,
+	}
+	err := ctx.store.CreateLoopIn(context.Background(), swapHash, swap)
+	require.NoError(t, err, "CreateLoopOut failed")
+
+	// Now read all the swaps from the store
+	swapInfos, err := ctx.swapClient.FetchSwaps(context.Background())
+	require.NoError(t, err, "FetchSwaps failed")
+
+	// Find the loop-in and check LastHop.
+	require.Len(t, swapInfos, 1)
+	loopInInfo := swapInfos[0]
+	require.Equal(t, &lastHop, loopInInfo.LastHop)
```

Comment:
nit: could recreate the expected swap info and check equality for the whole struct.

Reply:
+1

Reply:
Fixed